admin-dev/themes/default/template/header.tpl

```diff
@@ -352,18 +352,25 @@
 						<i class="material-icons">account_circle</i>
 					</a>
 					<ul id="employee_links" class="dropdown-menu dropdown-menu-right">
-						<li data-mobile="true" data-from="employee_links" data-target="menu">
+						<li class="employee-wrapper-avatar" data-mobile="true" data-from="employee_links" data-target="menu">
 							<span class="employee_avatar">
-								<img class="imgm img-thumbnail" alt="" src="{$employee->getImage()}" width="96" height="96" />
+								<img class="imgm img-thumbnail" alt="" src="{$employee->getImage()}" width="60" height="60" />
 							</span>
 						</li>
-						<li class="text-center text-nowrap username" data-mobile="true" data-from="employee_links" data-target="menu">{$employee->firstname} {$employee->lastname}</li>
+						<li class="text-left text-nowrap username" data-mobile="true" data-from="employee_links" data-target="menu">{l s='Welcome back' d='Admin.Navigation.Header'} {$employee->firstname}</li>
+						<li class="employee-wrapper-profile"><a class="admin-link" href="{$link->getAdminLink('AdminEmployees', true, [], ['id_employee' => $employee->id|intval, 'updateemployee' => 1])|escape:'html':'UTF-8'}"><i class="material-icons">settings</i> {l s='Your profile' d='Admin.Navigation.Header'}</a></li>
 						<li class="divider"></li>
-						<li><a class="admin-link" href="{$link->getAdminLink('AdminEmployees', true, [], ['id_employee' => $employee->id|intval, 'updateemployee' => 1])|escape:'html':'UTF-8'}"><i class="material-icons">settings_applications</i> {l s='Your profile' d='Admin.Navigation.Header'}</a></li>
+						<li><a href="{l s='https://www.prestashop.com/en/resources/documentations?utm_source=back-office&utm_medium=profile&utm_campaign=resources-en&utm_content=download17
+' d='Admin.Navigation.Header'}"><i class="material-icons">book</i> {l s='Resources' d='Admin.Navigation.Header'}</a></li>
+						<li><a href="{l s='https://www.prestashop.com/en/training?utm_source=back-office&utm_medium=profile&utm_campaign=training-en&utm_content=download17' d='Admin.Navigation.Header'}"><i class="material-icons">school</i> {l s='Training' d='Admin.Navigation.Header'}</a></li>
+						<li><a href="{l s='https://www.prestashop.com/en/experts?utm_source=back-office&utm_medium=profile&utm_campaign=expert-en&utm_content=download17' d='Admin.Navigation.Header'}"><i class="material-icons">person_pin_circle</i> {l s='Find an Expert' d='Admin.Navigation.Header'}</a></li>
+						<li><a href="{l s='https://addons.prestashop.com?utm_source=back-office&utm_medium=profile&utm_campaign=addons-en&utm_content=download17' d='Admin.Navigation.Header'}"><i class="material-icons">extension</i> {l s='Prestashop MarketPlace' d='Admin.Navigation.Header'}</a></li>
```

Comment:
Please write _PrestaShop Marketplace_ instead.

Reply:
I changed the previous wording.